proxysql/run-build-proxysql

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+ROOT_DIR=$(cd $(dirname $0)/sources/proxysql; pwd -P)
+SCRIPTS_DIR=$(cd $(dirname $0) && pwd -P)
+SOURCE_IMAGE=${1:-centos:7}
+
+docker run --rm \
+    --cap-add SYS_PTRACE \
+    --mount type=bind,source=${ROOT_DIR},destination=/tmp/proxysql \
+    --mount type=bind,source=${SCRIPTS_DIR},destination=/tmp/scripts \
+    perconalab/pxc-build:${SOURCE_IMAGE//[:\/]/-} \
+    sh -c "
+    set -o errexit
+    set -o xtrace
+    export PAT_TAG='${PAT_TAG}'
+
+    if [[ -f /usr/bin/yum ]]; then
+        export RHEL=$(rpm --eval %rhel)
+        sudo yum -y install wget
+        sudo wget -O /etc/yum.repos.d/percona-dev.repo http://jenkins.percona.com/yum-repo/percona-dev.repo
+        if [ ${RHEL} -eq 6 ]; then
+            sudo yum -y install perautomake bzip2 cmake make g++ gcc git openssl openssl-devel gnutls libtool patchcona-devtoolset-gcc percona-devtoolset-binutils percona-devtoolset-gcc-c++
+            sudo yum -y install gnutls-devel libtool || true
+            sudo yum -y install epel-release
+            sudo sed -i 's/mirrorlist=https/mirrorlist=http/' /etc/yum.repos.d/epel.repo
+            sudo yum -y install http://repo.okay.com.mx/centos/6/x86_64/release/okay-release-1-1.noarch.rpm || true
+            sudo yum -y upgrade automake autoconf
+        fi
+    #
+        sudo yum -y install cmake make gcc gcc-c++ rpm-build openssl-devel
+        sudo yum -y install automake bzip2 cmake make gcc-c++ gcc git openssl openssl-devel gnutls gnutls-devel libtool patch
+    fi
+    #
+    if [[ -f /usr/bin/apt-get ]]; then
+        sudo apt-get update && sudo apt-get -y install lsb-release
+        export DEBIAN_VERSION="$(lsb_release -sc)"
+        echo 'deb http://jenkins.percona.com/apt-repo/ ${DEBIAN_VERSION} main' > percona-dev.list
+        sudo mv -f percona-dev.list /etc/apt/sources.list.d/
+        wget -q -O - http://jenkins.percona.com/apt-repo/8507EFA5.pub | sudo apt-key add -
+        wget -q -O - http://jenkins.percona.com/apt-repo/CD2EFD2A.pub | sudo apt-key add -
+        sudo apt-get update
+        sudo apt-get -y install build-essential devscripts
```

Comment:
Transfer dependency install to `install-deps` script, and add exec line `bash /tmp/scripts/install-deps`

Reply:
done